src/main/java/com/sampleweb/pages/ComputersAndAccessoriesPage.java

```diff
@@ -0,0 +1,70 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class ComputersAndAccessoriesPage extends HomePage {
+    public String title = "Computers & Accessories";
+    private String pageTitle = "//*[@id=\"fst-hybrid-dynamic-h1\"]/div/h1/b";
+    private String globalStore = "//*[@id=\"leftNav\"]/ul[3]/div/li/span/span/div/label/span/span/img";
+    private String averageReviewsText = "//*[@id=\"leftNav\"]/h4[9]";
+    private String menuText = "//*[@id=\"nav-subnav\"]/a[1]";
+    private String dropDownMenu = "//*[@id=\"searchDropdownBox\"]";
+    public static String PATH = "https://www.amazon.co.uk/s?i=computers&ref=nb_sb_noss";
+    private String products = "//*[@class='s-result-list s-search-results sg-row']//*[@class='s-include-content-margin s-border-bottom']";
+    private String amazonPrimeCheckbox = "//*[@id=\"leftNav\"]/ul[5]/div/li[1]/span/span/div/label/input";
+    private String amazonPrimeLabel = "//*[@aria-label='Amazon Prime' or @class='a-size-base a-color-secondary']";
```

Comment:
Maybe you can declare these variables inside of a method when you need them? I don't think we need them all of them to be global.

Reply:
You could make the strings constant with final. Also having hardcoded indexes in the xpath eg ul[5] makes the code brittle and might make the tests fail if a new list element is added.

Reply:
these are better as private final, not sure why some are private and others public!